src/main/java/com/zufar/onlinestore/payment/service/PaymentGetter.java

```diff
@@ -0,0 +1,24 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.zufar.onlinestore.payment.dto.PaymentDetailsDto;
+import com.zufar.onlinestore.payment.exception.PaymentNotFoundException;
+import com.zufar.onlinestore.payment.mapper.PaymentConverter;
+import com.zufar.onlinestore.payment.repository.PaymentRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class PaymentGetter {
+
+    private final PaymentRepository paymentRepository;
+    private final PaymentConverter paymentConverter;
+
+    public PaymentDetailsDto getPayment(Long paymentId) {
```

Comment:
May be we can check if paymentId is null and throw exception?

Reply:
done